### ISSUE TITLE
Suitability check maps

### ISSUE THREAD

**Issue by cmahony** [whmacken/CCISS_JournalPaper#6]
@whmacken, 
here is a sample of the check maps is this what you were thinking? 

![EnvSuit Fd C4](https://user-images.githubusercontent.com/17104005/55831003-01ff2180-5ac7-11e9-9f02-5b40b7909e8a.png)

I made a map of the BGC units. note the units are labelled for close inspection. 

![BGCunits](https://user-images.githubusercontent.com/17104005/55831100-3541b080-5ac7-11e9-9b45-10e8772df4a4.png)

**Comment by whmacken** [comment]
Yes, this is about right. Is the BC portion modelled BGC or mapped BGC?

**Comment by cmahony** [comment]
modeled bgc.
